CommandLine/emake/EnigmaCallbacks.cpp

```diff
@@ -11,24 +11,24 @@ void* CallBack::OutputThread(void*)
 {
   if (_outFile)
   {
-    _outFile.seekg (0, _outFile.end);
-
-    int length = _outFile.tellg();
-
-    _outFile.seekg (0, _outFile.beg);
-
-    char * buffer = new char [length];
-
-    _outFile.read (buffer,length);
-
-    std::cout << buffer;
-
-    _outFile.close();
-
-    delete[] buffer;
+    int pos = 0;
+    while (!_outFile.eof()) {
+      _outFile.seekg(0, _outFile.end);
+      int length = (int)_outFile.tellg() - pos;
+      _outFile.seekg(pos, _outFile.beg);
+ 
+      if (length > 0) {
+		char* buffer = new char[length + 1];
```

Comment:
Why did you switch to tabs, here...?

Reply:
cause I wrote cli w/ spaces 2 years ago and forgot to change my editor back